src/Libraries/RevitNodes/Elements/FamilyInstance.cs

```diff
@@ -155,6 +155,9 @@ private void InitFamilyInstance(Autodesk.Revit.DB.FamilySymbol fs, XYZ pos)
 
         private void InternalSetLevel(Autodesk.Revit.DB.Level level)
         {
+            if (InternalFamilyInstance.LevelId.Compare(level.Id) == 0)
```

Comment:
No. For reasons described in the purpose section, the family instance is set to the same level twice.